### PR TITLE
infectivenesss to infectiousness

### PR DESCRIPTION
**Pull request by Elizabeth-Hayman**

### CODE REVIEW

**Comment by Elizabeth-Hayman** [comment]
Sticking with infectiousness in line with CovidSim. Haven't changed anything in the C++ backend

File: pyEpiabm/pyEpiabm/routine/covidsim_helpers.py [hunk 1]
```diff
@@ -15,13 +15,13 @@ class CovidsimHelpers:
...
     def calc_house_inf(infector: Person, timestep: int):
-        """Calculate the infectiveness of a household.
+        """Calculate the infectiousness of a household.
 
         :param infector: Infector
         :type infector: Person
         :param timestep: Current simulation timestep
         :type timestep: int
-        :return: Infectiveness parameter of household
+        :return: infectiousness parameter of household
```
Comment (KCGallagher) [inline pyEpiabm/pyEpiabm/routine/covidsim_helpers.py:24]:
```suggestion
        :return: Infectiousness parameter of household
```

**Review by KCGallagher — CHANGES_REQUESTED** [review]
I can't decide if I love or hate the name of this branch

**Review by KCGallagher — APPROVED** [review]

**Comment by KCGallagher** [comment]
> Sticking with infectiousness in line with CovidSim. Haven't changed anything in the C++ backend

@NicholasFan235 would it be okay with you if we also did this to keep things consistent? Happy to do it myself off your latest version to avoid conflicts

**Comment by KCGallagher** [comment]
This commit contains the same changes of the cEpiabm backend  - can be easily reverted if we don't want it but hasn't broken anything!

**Pull request merged**